EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||e2wysbacctt1.com^$third-party
```

Comment:
this domain is already blocked by the base filter (easylist)

Reply:
Fixed.